### PR TITLE
add support for Z index

### PR DESCRIPTION
**Pull request by davidblasby**

### CODE REVIEW

File: bridgestyle/sld/fromgeostyler.py [hunk 1]
```diff
@@ -27,32 +59,30 @@ def convert(geostyler):
...
+            featureTypeStyle.append(processRule(rule))
+        if "blendMode" in geostyler:
+            _addVendorOption(featureTypeStyle, "composite", geostyler["blendMode"])
         
     sldstring = ElementTree.tostring(root, encoding='utf8', method='xml').decode()
-    dom = minidom.parseString(sldstring)    
-    return dom.toprettyxml(indent="  "), _warnings
+    dom = minidom.parseString(sldstring)
+    result = dom.toprettyxml(indent="  "), _warnings
+    return result
```
Comment (volaya) [inline bridgestyle/sld/fromgeostyler.py]: The result is expected to be a tuple, not just the xml

File: bridgestyle/sld/fromgeostyler.py [hunk 2]
```diff
@@ -8,6 +8,35 @@
...
+#     rule[0]["symbolizers"][1] has Z=1
+#
+# this will return
+#   result[0] => rule with symbolizer 0  (name changed to include Z=0)
+#   result[1] => rule with symbolizer 1  (name changed to include Z=1)
+def processRulesByZ(rules):
+    result = {}
+    for rule in rules:
+        for symbolizer in rule["symbolizers"]:
+            z = 0
```
Comment (volaya) [inline bridgestyle/sld/fromgeostyler.py]: these lines can be replaced with `z = symbolizer.get("Z", 0)`

File: bridgestyle/sld/fromgeostyler.py (hunk 1 above)
Comment (davidblasby) [reply bridgestyle/sld/fromgeostyler.py]: It does return the tuple - just as before.  I just put the result in a variable so I could see it more easily in the debugger.

File: bridgestyle/sld/fromgeostyler.py (hunk 2 above)
Comment (davidblasby) [reply bridgestyle/sld/fromgeostyler.py]: updated

**Pull request merged**